File: PASCAL/Cap04/c04ex03.pas
program c04ex03;

var
  A, B, X, R: Integer;

begin
  Write('Entre o 1o. valor numerico: ');
  ReadLn(A);

  Write('Entre o 2o. valor numerico: ');
  ReadLn(B);

  X := A + B;

  if (X >= 10) then
    R := X + 5
  else
    R := X - 7;

  WriteLn('Resultado = ', R);
  WriteLn;

  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: LUA/Cap04/c04ex03.lua
-- c04ex03.lua

io.write("Entre o 1o. valor numerico: ")
a = tonumber(io.read())

io.write("Entre o 2o. valor numerico: ")
b = tonumber(io.read())

x = a + b

if (x >= 10) then
  r = x + 5
else
  r = x - 7
end

print("Resultado = " .. r)

print()
io.write("Tecle <Enter> para encerrar... ")
io.read()
